profiles/license_groups

```diff
@@ -71,7 +71,7 @@ BINARY-REDISTRIBUTABLE @FREE bh-luxi Broadcom Dina intel-ucode ipw2100-fw ipw220
 
 # License agreements that try to take away your rights. These are more
 # restrictive than "all-rights-reserved" or require explicit approval.
-EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER-2 DOOM3 ETQW f.lux FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA geekbench genymotion GIMPS GOG-EULA google-chrome Google-TOS Intel-SDP Introversion LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms Microsemi MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge protonmail-bridge-EULA PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs StarUML-EULA supermicro teamspeak3 TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi worklog-assistant zi-labone
+EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER-2 DOOM3 ETQW f.lux FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA geekbench genymotion GIMPS GOG-EULA google-chrome Google-TOS Intel-SDP Introversion Kryoflux-MAME LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms Microsemi MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge protonmail-bridge-EULA PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs StarUML-EULA supermicro teamspeak3 TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi worklog-assistant zi-labone
```

Comment:
Why do you think this is an EULA?

Reply:
The included license sounds for me like a typical EULA, how it's written. Am I wrong?

Reply:
@gentoo/licenses, could you explain, please?

Reply:
&lt;mgorny&gt; ulm: on another topic, does a license forbidding 'use in commercial product' count as EULA?
&lt;mgorny&gt; (after all, i suppose that's more restrictive than all-rights-reserved)
&lt;ulm&gt; depends on the exact wording
&lt;ulm&gt; and if the user is forced to accept the license
&lt;ulm&gt; if it's otherwise redistributable then such a clause is normally moot
&lt;mgorny&gt; https://github.com/gentoo/gentoo/pull/7612/files#diff-551080d2cc01e3552ea8affa6a939771
&lt;mgorny&gt; he added it to @EULA but i think it's wrong
&lt;ulm&gt; bah, long lines :(
&lt;ulm&gt; but that doesn't look like an eula to me
[...]
&lt;ulm&gt; it's an EULA only if it forces the user into a contract
&lt;ulm&gt; in my understanding (as a non-lawyer) EULAs aren't even licenses

Reply:
s/@eula/@EULA/

Reply:
Allright, thanks for the evaluation! 👍 
I dropped it by now.